en/docs/references/ballerina-expression-syntax.md

```diff
@@ -2,421 +2,523 @@
+    
+!!! info
+    The examples in this section will assume the following json is already defined.
+    
+    ```ballerina
+         json person = {
+             fname: "Alex",
+             lname: "Jane",
+             address: {
+                 line: "20 Palm Grove",
+                 city: "Colombo 03",
+                 country: "Sri Lanka"
+             }
+         }
+    ```    
+
+- Field access(.) 
+
+    You can use the field access operator to access record fields of the `json` type. If the key is invalid or the field access occurs on `null`, it returns `error`.
+   
+    ###### Examples:
+
+    To access a field, use the following:    
+
+    ```ballerina
+       json|error firstName = person.fname;
+    ```
+    To access a field in a nested level, use the following:
+ 
+     ```ballerina
+        json|error city = person.address.city;
+     ```
 
-Sometimes, content is sent as a string (plain text) with the HTTP request. The `request` variable’s `getTextPayload` function can be used to read the text payload sent with the HTTP request. Note that the `getTextPayload` function’s return type is `string|ClientError`.
+- Optional field access (?.) 
 
-#### Example
+    You can use the optional field access operator to access record fields, including any optional fields. If the key is invalid or the field access occurs on `null`, it returns `null ` or `()`.
 
-- Reading the Text Payload of the HTTP request.
+    ```ballerina
+    json|error year = j1?.age?.year;
+    ```
 
-  ```ballerina
-  check request.getTextPayload()
-  ```
+For more information, refer to [Ballerina JSON Access](https://ballerina.io/learn/by-example/json-access.html)
+ 
+### JSON to XML
 
-### Reading the query parameter value
+You can convert JSON to using the `xmldata:fromJson()` function. This function takes a `xmldata:JsonOptions` record as an argument. The `JsonOptions` record has two fields: the `attributePrefix` and the `arrayEntryTag`. 
+ - `attributePrefix`: A prefix that you use to identify the XML attributes. The default value of the `attributePrefix` option is `@`. 
+ - `arrayEntryTag`: The name of the tag you add to each entry of the JSON array. The default value of the arrayEntryTag option is `root`.
 
-Query parameters are used in an HTTP request to send additional inputs to consider when processing the request. You can use the `request` variable's `getQueryParamValue` function to read the passed query parameter.
+!!! info
+    The examples in this section will assume the following json is already defined.
+    
+    ```ballerina
+    json storeDetails = {
+         "Store": {
+             "@id": "AST",
+             "name": "Alex
+             "address": {
+                 "street": "Main",
+                 "city": "94"
+             },
+             "codes": ["4", "8"]
+         }
+    };
+    ``` 
```

Comment:
Will code snippets display properly within info. Github  view is not  doing that.

Reply:
![Screenshot from 2021-06-04 11-48-03](https://user-images.githubusercontent.com/7173347/120754800-c63b3780-c52a-11eb-8a7b-18d79c253d6d.png)
The compiled view seems fine.